packages/kbn-expandable-flyout/src/hooks/use_preview.ts

```diff
@@ -0,0 +1,18 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import { rootKey } from './root_key';
+import { useUrlState } from './use_url_state';
+
+type PreviewState = any;
+
+export const usePreview = () => {
+  const [previewState, setPreviewState] = useUrlState(rootKey, 'preview');
```

Comment:
same here for the key, use `previewPanel` instead of `preview`?

Reply:
fixed, thanks!